bootstrap/index.js

```diff
@@ -49,8 +50,15 @@ const withLogger = (execute) => async (data) => {
   return result
 }
 
+const { toAsync } = util
+const { isAsyncFunction } = types
```

Comment:
I would say this is excessive destructuring, its hard to know what the parent module is when this is done too much

Reply:
Agreed! 💯